elasticsearch-persistence/test/integration/model/model_basic_test.rb

```diff
@@ -227,7 +228,17 @@ class ::Person
           assert_contains @results, 'John 1'
         end
       end
-
+      context "with sti" do
+        class ::Worker < ::Person
+          document_type 'human_being'
+          index_name 'people'
+        end
+        should "return a concrete class if a class type is stored" do
+          Worker.create name: "John", type: "worker"
+          Person.gateway.refresh_index!
+          assert_equal Worker, Person.all.first.class
```

Comment:
Does rails do the same when calling `Person.all`, i.e. return a mix of `Worker`s and other classes? Or is it Just `Person`s if you call `Person.all`?